### DISCUSSION TITLE
Looking for extensions for file editing

### DISCUSSION THREAD

**Discussion by Susja** [microsoft/vscode-discussions#801] [Extension Show and Tell]
I started to use on my MacBook Visual Studio Code and need to edit text files. Most of the file types I'm interested are .txt and .csv Before I used Notepad++ when I needed to do some editing like split file vertically , replacements using regular expression and etc I see a lot of extensions in VS Code but which one would you recommend .. quite lightweight but still remind Notepad++ or etc.? 
Note: my goal is not to split the file but rather be able to select vertical column and copy/paste
It looks that built-in features don’t allow vertical selection 
Thanks

**Comment by gjsjohnmurray** [comment]
What about https://code.visualstudio.com/docs/editor/codebasics#_column-box-selection?

**Comment by Susja** [comment]
Thanks ,
I will take a look,
Appreciate it

**Comment by Susja** [comment]
Hey John,
It looks like exactly I was looking for.
But unfortunately for some reason it doesn’t work for me
I am using MacBook.
I am using Azure Data Studio 1.45.1
In document you pointed there are a lot of combinations but none of It
works working for me. Sounds like I am doing something wrong.
Look… I have a text file with two columns separated by white space
My goal is to select only one column, copy it and paste somewhere.
I went to Settings- User and Workspace- Tex Editor- Multi Cursor Modifier
and set it to ctrlCmd
Then I went to File- Preferences- Keyboard Shortcuts and then selected
Toggle Multi-Cursor Modifier to Ctrl + Shift
Eventually I open a text file and place cursor at the beginning of the
first column I.e. top-left position and select simultaneously Ctrl + Shift
and start to move the mouse to make a selection but it doesn’t work.
What I am doing wrong?
Thanks and appreciate your help

**Reply by gjsjohnmurray** [reply]
If you use Command Palette to run `Toggle Column Selection Mode` do you get the behaviour you want, plus an indicator near the right-hand end of the status bar?

**Comment by Susja** [comment]
Perfect,
It did exactly what I was looking for!
I tried keybinding it too ... but it didn't work. Anyway ... I'm OK with
using Command Palette only.
Just curious ... Do you use key binding or Command Palette or both?

Thanks

On Thu, Sep 7, 2023 at 11:54 AM John Murray ***@***.***>
wrote:

> If you use Command Palette to run Toggle Column Selection Mode do you get
> the behaviour you want, plus an indicator near the right-hand end of the
> status bar?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/microsoft/vscode-discussions/discussions/801#discussioncomment-6939118>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AGJMF7QFZZ5VKNPRAWYUXXDXZHU2LANCNFSM6AAAAAA4NEPS6Q>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> .com>
>


-- 
<https://mail.google.com/mail/u/0/?ui=2&ik=d3981d59f9&view=att&th=136febab6da21f8f&attid=0.0.1&disp=emb&realattid=ii_136d59273e294831&zw&atsh=1>
Eliya Voldman

**Reply by gjsjohnmurray** [reply]
I hardly ever need to do Column Select, so I'd almost certainly use Command Palette.

**Reply by Susja** [reply]
Thanks so much 
I am all set now
